tests/integration/test_notebooks_python.py

```diff
@@ -196,6 +196,30 @@ def test_wikidata_integration(notebooks, tmp):
     assert results["length_result"] >= 1
 
 
+@pytest.mark.integration
```

Comment:
since you import torch in the notebook you will need to add a gpu marker
@pytest.mark.gpu

also, I just pushed some restructuring to the tests folders, so please merge staging into your branch and move this test into tests/integration/examples/test_notebook_gpu.py

Reply:
same thing applies to the other notebooks of course =)

Reply:
fixed!